cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -101,6 +101,56 @@ module cosmos 'br/public:avm/res/document-db/database-account:0.8.1' = {
 }
 {{- end}}
 
+{{- if .DbCosmos }}
+module cosmos 'br/public:avm/res/document-db/database-account:0.8.1' = {
+  name: 'cosmos'
+  params: {
+    name: '${abbrs.documentDBDatabaseAccounts}${resourceToken}'
```

Comment:
We need a unique name here -- right now, it collides with `cosmosMongo` above. I'm not sure yet what a better naming strategy is.. Open to suggestions.

Reply:
If I am reading this correctly, Cosmos DB doesn't allow multilple APIs per database account, so we need to create 2

Reply:
I will change the previous one to `cosmosMongo`

Reply:
Yes, you are correct.

Reply:
I pushed a change to update the existing mongoDB abberviation. It's a breaking change, but we're okay given the alpha stage.